benchmark_runner/workloads/workloads_operations.py

```diff
@@ -524,6 +533,8 @@ def initialize_workload(self):
         self._template.generate_yamls(scale=str(self._scale), scale_nodes=self._scale_node_list, redis=self._redis, thread_limit=self._threads_limit)
         if self._enable_prometheus_snapshot:
             self.start_prometheus()
+        if self._google_drive_path:
+            self.find_or_create_run_artifacts_google_drive()
```

Comment:
find_or_create_run_artifacts_drive returns the new drive, but this doesn't use the returned value.

Reply:
I revert it back to create